### ISSUE TITLE
Add plotting script for benchmarks from files 

### ISSUE THREAD

**Issue by phylyc** [IBM/pde-deep-learning#5]
The benchmark data is in a tab separated text file with column names
date	seed	mesh_size	tile	n_instances	n_input	n_classes	learning_rate	reg_coeff	batch_size	epochs	cc_coeff	kappa	n_layers	n_nodes	iter	MSE	MAE	MAPE	sMAPE	training_time	ML_execution_time

A small script is needed that can plot and compare several benchmarks
